.github/workflows/e2e.yml

```
@@ -111,6 +111,20 @@ jobs:
     secrets:
       DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
       DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+  
+  e2e-leg-1-admintools-previous-release:
+    if: ${{ inputs.e2e_test_suites == 'all' || inputs.e2e_test_suites == 'admintools leg 1' || inputs.e2e_test_suites == '' }}
+    needs: [build] 
+    uses: ./.github/workflows/e2e-leg-1.yml
+    with:
+      vlogger-image: ${{ needs.build.outputs.vlogger-image }}
+      operator-image: ${{ needs.build.outputs.operator-image }}
+      vertica-image: "vertica/vertica-k8s:23.4.0-0-minimal"
+      artifact-version: "23.4.0-release"
```

Suggest changing this if you accept my other comment:
```suggestion
      artifact-suffix: "-23.4.0-release"
```